pkg/util/strategicpatch/patch.go

```diff
@@ -523,13 +523,13 @@ func getTagStructType(dataStruct interface{}) (reflect.Type, error) {
 
 var errBadPatchTypeFmt = "unknown patch type: %s in map: %v"
 
-// Merge fields from a patch map into the original map. Note: This may modify
+// MergeMap merges fields from a patch map into the original map. Note: This may modify
 // both the original map and the patch because getting a deep copy of a map in
```

Comment:
Is this still true?  Deep copy of a map isn't that hard for `map[string]interface{}`